From ae211a3fcfb1b827f0ec35d61f28d6b539028812 Mon Sep 17 00:00:00 2001
From: Till Kamppeter <till.kamppeter@gmail.com>
Date: Wed, 3 Oct 2018 15:13:00 +0200
Subject: Fixed incomplete removal of hp-toolbox features, which caused the
 hp-toolbox not to start

Looking into the ui5/devmgr5.py of the HPLIP Python source code it
seems that some features got removed by commenting them out, but this
removal seems not to have been completed, making non-existing objects
being called during the start of hp-toolbox and so it crashes right
away.

This patch completes the commenting-out and so makes hp-toolbox start
up and work again.

Bug-Debian: https://bugs.debian.org/912768
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1789184
---
 ui5/devmgr5.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/ui5/devmgr5.py b/ui5/devmgr5.py
index 0ca016820..9d4e82c58 100644
--- a/ui5/devmgr5.py
+++ b/ui5/devmgr5.py
@@ -319,11 +319,11 @@ class DevMgr5(Ui_MainWindow_Derived, Ui_MainWindow, QMainWindow):
         self.PreferencesAction.setIcon(QIcon(load_pixmap('settings', '16x16')))
         self.PreferencesAction.triggered.connect(self.PreferencesAction_activated)
 
-        self.DiagnoseQueueAction.setIcon(QIcon(load_pixmap('warning', '16x16')))
-        self.DiagnoseQueueAction.triggered.connect(self.DiagnoseQueueAction_activated)
+        #self.DiagnoseQueueAction.setIcon(QIcon(load_pixmap('warning', '16x16')))
+        #self.DiagnoseQueueAction.triggered.connect(self.DiagnoseQueueAction_activated)
 
-        self.DiagnoseHPLIPAction.setIcon(QIcon(load_pixmap('troubleshoot', '16x16')))
-        self.DiagnoseHPLIPAction.triggered.connect(self.DiagnoseHPLIP_activated)
+        #self.DiagnoseHPLIPAction.setIcon(QIcon(load_pixmap('troubleshoot', '16x16')))
+        #self.DiagnoseHPLIPAction.triggered.connect(self.DiagnoseHPLIP_activated)
 
         self.ContentsAction.setIcon(QIcon(load_pixmap("help", "16x16")))
         self.ContentsAction.triggered.connect(self.helpContents)
